src/test/java/com/incognia/TokenAwareDispatcher.java

```diff
@@ -43,6 +45,10 @@ public MockResponse dispatch(@NotNull RecordedRequest request) {
       assertThat(request.getHeader("Authorization")).isEqualTo("Bearer " + token);
       return new MockResponse().setResponseCode(200).setBody("{\"name\": \"my awesome name\"}");
     }
+    if ("/api/v2/onboarding".equals(request.getPath()) && "GET".equals(request.getMethod())) {
+      assertThat(request.getHeader("Authorization")).isEqualTo("Bearer " + token);
+      return new MockResponse().setResponseCode(200).setBody("{\"name\": \"my awesome name\"}");
```

Comment:
I think we can refactor these body strings and paths to constants

Reply:
Also i think that this method is getting a little bit out of hand with so much code :smile:
Do you think this is the right time to refactor the content of these ifs with methods? or should we wait a little more?

Reply:
I guess I can refactor this in another PR